src/main/resources/db/migration/V0.2/V0.2.4__revise_cascading_deletes.sql

```diff
@@ -0,0 +1,30 @@
+-- change some ON DELETE CASCADE clauses to ON DELETE SET NULL for relationships whose referenced objects might be needed elsewhere even when a related object is deleted
```

Comment:
Don't know what I was thinking when I worked on the customer account story, but I ended up thinking about a lot of the cascades backwards; I had thought I was setting things up so that if (for example) a `customer_account` was deleted, its addresses and phone numbers would be deleted as well, but I actually had it set up so that if one of the phone numbers or addresses of the `customer_account` would be deleted, the entire `customer_account` would be deleted as well. This file fixes that.